util/response_request.py

```diff
@@ -0,0 +1,30 @@
+from util.str_utils import preprocess_text
+import requests
+
+
+def request_hero_name_set():
+    WEB_REQUEST = requests.get('https://dota2.gamepedia.com/api.php?' +
```

Comment:
As a general rule of thumb, variables names shouldn't be uppercase unless they are constant and final (or atleast what I follow in Java for static final variables). Variables used only for inner method logic should be in snake_case.

Reply:
Fixed in https://github.com/Jonarzz/DotaResponsesRedditBot/pull/41/commits/c370f726561bfe9b60a2ec63a0d868aa89905b8a.